pkg/runtime/podman/podman.go

```
@@ -140,6 +149,22 @@ func (p *podmanRuntime) cleanupPodFiles(containerID string) {
 	os.RemoveAll(p.podDir(containerID))
 }
 
+const podTemplateDataFile = "pod-template-data.json"
+
+// readPodTemplateData reads the persisted pod template data for a container.
+func (p *podmanRuntime) readPodTemplateData(containerID string) (podTemplateData, error) {
```

golangci-lint (2.11.4)

[error] func (*podmanRuntime).readPodTemplateData is unused

(unused)